_examples/finally/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+// Example application using the github.com/briandowns/spinner API
+
+import (
+	"time"
+
+	"github.com/briandowns/spinner"
+)
+
+func main() {
+	s := spinner.New(spinner.CharSets[9], 100*time.Millisecond) // Build our new spinner
+	s.Color("red")                                              // Set the spinner color to red
+	s.Finally = "Complete!\nNew line!\nAnother one!\n"
```

Comment:
Not near my computer at the moment but I think I updated the mistakes there. I'll have to check. 

> On Feb 27, 2016, at 18:16, Jacob Evans <notifications@github.com> wrote:
> 
> In _examples/finally/main.go:
> 
> > @@ -0,0 +1,18 @@
> > +package main
> > +
> > +// Example application using the github.com/briandowns/spinner API
> > +
> > +import (
> > +	"time"
> > +
> > +	"github.com/briandowns/spinner"
> > +)
> > +
> > +func main() {
> > +	s := spinner.New(spinner.CharSets[9], 100*time.Millisecond) // Build our new spinner
> > +	s.Color("red")                                              // Set the spinner color to red
> > +	s.Finally = "Complete!\nNew line!\nAnother one!\n"
> Should this be:
> 
> s.FinalMSG = "Complete!\nNew line!\nAnother one!\n"
> Finally doesn't seem to be In the structure
> 
> —
> Reply to this email directly or view it on GitHub.
>